frontend/resourceadm/components/PartyListDetails/PartyListDetail.test.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render as rtlRender, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { textMock } from '../../../testing/mocks/i18nMock';
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+import { PartyListDetail, PartyListDetailProps } from './PartyListDetail';
+import { ServicesContextProps, ServicesContextProvider } from 'app-shared/contexts/ServicesContext';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+
+const mockedNavigate = jest.fn();
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedNavigate,
+}));
+
+const testOrg = 'ttd';
+const testEnv = 'tt02';
+const testListeIdentifier = 'listid';
+
+const defaultProps = {
+  org: testOrg,
+  env: testEnv,
+  list: {
+    env: testEnv,
+    identifier: testListeIdentifier,
+    name: 'Test-liste',
+    description: 'Dette er en beskrivelse',
+    members: [
+      {
+        orgNr: '123456789',
+        orgName: '',
+        isUnderenhet: false,
+      },
+    ],
+  },
+  backUrl: '/listadmin',
+};
+
+const user = userEvent.setup();
```

Comment:
This should be defined inside each of the it blocks as [recommended by the testing-library](https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent) 😄